### PR TITLE
added Pereira-2018-180

### PR DESCRIPTION
**Pull request by Kristina-Pianykh**
#605. This is quite an unusual list as it isn't based on any other lists but rather was compiled from scratch. As you can see, there're quite a few unmapped concepts due to their semantic and grammatical specificity.

Since there were some glosses on concepticon, identical to those in Pereira's list and also unmapped, I created new concept sets for them.

I also left unmapped _hurting_ and _residence_ because of their ambiguity which the authors don't comment on.

I'm not quite sure, however, if _jungle_ should be mapped to FOREST (as it is in [Marrison-1967-917](https://concepticon.clld.org/contributions/Marrison-1967-917)). What do you think?

### CODE REVIEW

File: concepticondata/concepticon.tsv [hunk 1]
```diff
@@ -3449,3 +3449,10 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3463	RIVERBANK	The physical world	The margin or shore of a river.	Person/Thing	
 3464	SHORE OF LAKE	The physical world	The margin or shore of a lake.	Person/Thing	
 3465	SHORE OF SEA	The physical world	The margin or shore of a sea or ocean.	Person/Thing	
+3466	DANGEROUS	Emotions and values	Capable of injuring or harming someone.	Property	
+3467	HELP (AID)	Social and political relations	The act of helping someone.	Person/Thing
```
Comment (LinguList) [inline concepticondata/concepticon.tsv]: maybe HELP (ACTION) as gloss?

File: concepticondata/concepticon.tsv [hunk 2]
```diff
@@ -3449,3 +3449,10 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3463	RIVERBANK	The physical world	The margin or shore of a river.	Person/Thing	
 3464	SHORE OF LAKE	The physical world	The margin or shore of a lake.	Person/Thing	
 3465	SHORE OF SEA	The physical world	The margin or shore of a sea or ocean.	Person/Thing	
+3466	DANGEROUS	Emotions and values	Capable of injuring or harming someone.	Property	
+3467	HELP (AID)	Social and political relations	The act of helping someone.	Person/Thing	
+3468	KINDNESS	Emotions and values	The quality of being caring, generous and helpful towards other people.	Property	
+3469	LEVEL	Quantity	The amount or degree of something; the height of the surface of something liquid in respect to its bottom.	Person/Thing
```
Comment (LinguList) [inline concepticondata/concepticon.tsv]: I'm not sure with resp. to the English: can you say `in respect to`? If so, ignore this remark.

File: concepticondata/concepticon.tsv [hunk 3]
```diff
@@ -3449,3 +3449,10 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3463	RIVERBANK	The physical world	The margin or shore of a river.	Person/Thing	
 3464	SHORE OF LAKE	The physical world	The margin or shore of a lake.	Person/Thing	
 3465	SHORE OF SEA	The physical world	The margin or shore of a sea or ocean.	Person/Thing	
+3466	DANGEROUS	Emotions and values	Capable of injuring or harming someone.	Property	
+3467	HELP (AID)	Social and political relations	The act of helping someone.	Person/Thing	
+3468	KINDNESS	Emotions and values	The quality of being caring, generous and helpful towards other people.	Property	
+3469	LEVEL	Quantity	The amount or degree of something; the height of the surface of something liquid in respect to its bottom.	Person/Thing	
+3470	LIAR	Social and political relations	The person who tells lies.	Person/Thing
```
Comment (LinguList) [inline concepticondata/concepticon.tsv]: `The person` -> `A person`. These constructions require indefinite article usually.

File: concepticondata/concepticon.tsv (hunk 1 above)
Comment (LinguList) [reply concepticondata/concepticon.tsv]: A, sorry: not HELP (ACTION), but rather HELP (ASSISTANCE)? OUr convention is not clear here, but you'd like to have a real noun in brackets to indicate it's a noun. And assistance was the first hit on a dictionary I found.

File: concepticondata/conceptlists.tsv [hunk 4]
```diff
@@ -258,4 +258,5 @@ Backstrom-1992-210d	Backstrom, Peter C. and Radloff, Carla F.	1992	d	201	basic,a
...
+Pereira-2018-180	Pereira, Francisco and Lou, Bin and Pritchett, Brianna and Ritter, Samuel and Gershman, Samuel J. and Kanwisher, Nancy and Botvinick, Matthew and Fedorenko, Evelina	2018		180	specific	English	English	https://www.nature.com/articles/s41467-018-03068-4	Pereira2018		The given list was compiled in the frameworks of research on the neural basis of semantic representations. As reported by the authors, it resulted from clustering words from [the massive text corpora](:bib:Brysbaert2014) into 180 semantic groups, covering the entire semantic space. The most representative concepts, manually selected from each cluster, were then used as input stimuli in three fMRI experiments in order to investigate the relationship between the linguistic representations and the resulting imaging data. The researchers report that semantic information is not confined to a specific brain region, as commonly assumed, but rather distributed across several high-level cortical networks.	1-13
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv]: `[the massive text corpora]` -> `[massive text corpora]` no article here. `the entire semantic space` -> `the entire semantic space of the corpus`.

File: concepticondata/conceptlists/Pereira-2018-180.tsv [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+Pereira-2018-180-38	38	counting	2	verb	1420	COUNT
+Pereira-2018-180-39	39	crazy	3	adj	1899	MAD
+Pereira-2018-180-40	40	damage	1	noun	2947	DAMAGE (INJURY)
+Pereira-2018-180-41	41	dance	1	noun	3472	DANCE (MOVEMENT)
+Pereira-2018-180-42	42	dangerous	3	adj	3466	DANGEROUS
+Pereira-2018-180-43	43	deceive	2	verb	3230	DECEIVE
+Pereira-2018-180-44	44	dedication	1	noun		
+Pereira-2018-180-45	45	deliberately	4	adv		
+Pereira-2018-180-46	46	delivery	1	noun		
+Pereira-2018-180-47	47	dessert	1	noun
```
Comment (LinguList) [inline concepticondata/conceptlists/Pereira-2018-180.tsv]: OMG our concepticon lacks a concept for `dessert`. That disqualifies us for any merchandising enterprises with Nestlé! Sorry, just find it interesting to note how much what is "basic" seems to have changed. Ignore this comment.

File: concepticondata/concepticon.tsv (hunk 2 above)
Comment (chrzyki) [reply concepticondata/concepticon.tsv]: In respect of or with respect to

**Pull request merged**